components/Harvesters/AdminHarvestersPage.vue

```diff
@@ -85,7 +85,7 @@
             >
               <td>
                 <AdminContentWithTooltip>
-                  <NuxtLinkLocale
```

Comment:
There is no href fo these links so they are disabled

Reply:
Fixed in b07c1b6